.gitignore

```diff
@@ -7,6 +7,7 @@ dropin.cache
 twistd.pid
 docs/_build
 _tmp
-_trial_temp
-.tox
+_trial_temp*
+.tox*
 *.egg-info
+*.sw?
```

Comment:
Could these configuration changes be split out as well?  They seem pretty independent.